README.md

```diff
@@ -1 +1,81 @@
-# th2-conn-ws-client
\ No newline at end of file
+# WebSocket Client
+
+This microservice allows sending and receiving of messages via WebSocket protocol
+
+## Configuration
+
+Main configuration is done via setting following properties in a custom configuration:
```

Comment:
**The main** configuration is done via **settings** following **the** properties in a custom configuration:

Reply:
Why `settings following the properties`? In original sentence I've meant to say that in order to change configuration a user have to change values of properties described below

Reply:
Ok, I understood now. In this case, please disconsider my comment above.